scripts/mods/msu/popup.nut

```diff
@@ -1,10 +1,15 @@
 this.popup <- {
+	State = {
+		None = 0,
+		Small = 1,
+		Full = 2
+	}
 	m = {
 		Visible = false,
 		Animating = false,
 		JSHandle = null,
-		TextCache = "",
-		ForceQuit = false
+		ForceQuit = false,
+		FunctionBuffer = [],
```

Comment:
Yeah I'm not sure why I called it a cache, buffer is the correct term, I like the change here.